src/main/java/org/folio/gobi/Mapper.java

```diff
@@ -555,11 +555,6 @@ private void mapEresource(List<CompletableFuture<?>> futures, Eresource eresourc
         .thenAccept(o -> eresource.setActivated((Boolean) o))
         .exceptionally(Mapper::logException)));
 
-    Optional.ofNullable(mappings.get(Mapping.Field.CREATE_INVENTORY))
```

Comment:
> Make necessary adjustments to allow for these fields to be mapped if desired.
According to the story we would have to still provide the ability to map this field if required by the tenant, so this field has to be mapped in this class for both physical and eresource.

Reply:
please, take a look a the last updates